iiasa/message_ix
# Add new logo and diagram to Sphinx/RTD docs

This PR adds the new `message-ix` logo and diagram to the documentation and is the corresponding PR for https://github.com/iiasa/message_data/pull/368 opened by @khaeru.

## How to review

- Read the diff and note that the CI checks all pass.
- Run a specific code snippet or command and check the output.
- View the RTD docs preview at **https://docs.messageix.org/en/feature-new_logo/** look at:
  - the combined IIASA-`message-ix` logo top left,
  - the favicon, and
  - the new diagram under "Getting started".

## PR checklist
